bitnami/airflow/README.md

```diff
@@ -216,6 +216,12 @@ externalDatabase.existingSecret=db-secret
 externalDatabase.existingSecretSqlConnectionKey=sql-connection
 ```
 
+### Database setup
+
+By default, this chart setups the database (init or migrate the schema) and creates the admin user using a K8s job that is created when the chart release is installed or upgraded, and deleted once it success. This job uses [Chart hooks](https://helm.sh/docs/topics/charts_hooks), so it won't be deleted if you're using Helm exclusively for its rendering capabilities (e.g. when using ArgoCD or FluxCD).
```

Comment:
Once it succeeds

Reply:
done at https://github.com/bitnami/charts/pull/30440/commits/0d55c99d195fdf3277bb6efc87702264769e9a9e